### PR TITLE
Add support for LaTeX templates

### PR DESCRIPTION
**Pull request by chrisjsewell**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,93 +1,155 @@
...
-        "path": "./syntaxes/jinja-yaml.tmLanguage.json"
-      },
-      {
-        "language": "jinja",
-        "scopeName": "source.jinja",
-        "path": "./syntaxes/jinja.tmLanguage.json"
-      }
-    ],
-    "snippets": [
+	"name": "jinjahtml",
```
Comment (samuelcolvin) [inline package.json]: Please could you remove this change, keep indents with 2 spaces.

Comment (chrisjsewell) [reply package.json]: Hey yeh no problem, that should be fixed now. I notice that GitHub isn't doing a very good job at diffing json but, when you compare them in vscode, they are now exactly the same, except for the sections relevant to latex (and a version bump).

Comment (samuelcolvin) [reply package.json]: you version of `package.json` is still indented with 4 spaces. vscode does some very "clever" (mostly useful, occasionally confusing) things with ignoring indent-only changes when showing changes, but the file is still changed.

Please indent `package.json` with 2 spaces. Use `git diff` if you're unsure.

File: package.json [hunk 2]
```diff
@@ -76,18 +83,24 @@
...
       }
     ],
     "snippets": [
-			{
-				"language": "jinja-html",
-				"scopeName": "text.html.jinja",
-				"path": "./snippets/django-snippets.json"
-			}
-		]
+                      {
```
Comment (samuelcolvin) [inline package.json]: This is still indented with like a million spaces. :smile:

Comment (chrisjsewell) [reply package.json]: Yeh so weird, the indentation looked totally different looking at it on here. But, sorted I hope this time!

File: syntaxes/jinja-latex.tmLanguage.json [hunk 3]
```diff
@@ -0,0 +1,342 @@
...
+      "captures": [
+        {
+          "name": "entity.other.jinja.delimiter.comment"
+        }
+      ],
+      "end": "-?#\\)\\)",
+      "name": "comment.block.jinja"
+    },
+    {
+      "begin": "\\(\\(\\(-?",
```
[the remark this answers is not in the document]
Comment (chrisjsewell) [reply syntaxes/jinja-latex.tmLanguage.json]: Yep, that's for variable printing, an exemplar of the format this is based on is here: https://github.com/jupyter/nbconvert/blob/master/nbconvert/templates/latex/base.tplx

**Comment by samuelcolvin** [comment]
Thanks a lot. Much appreciated. Should be deployed now.

Would be great if you could [review](https://marketplace.visualstudio.com/items?itemName=samuelcolvin.jinjahtml) the extension. Some idiot couldn't work out how to use it and gave it 3 stars, thus lowering the star rating.

**Comment by chrisjsewell** [comment]
Done, cheers for the package :)

**Comment by samuelcolvin** [comment]
Just seen it, thanks a lot.

**Pull request merged**